_includes/list.html

```diff
@@ -18,7 +18,9 @@
                 {% endfor %}
             {% endfor %}
         </div>
-        <p class="divider footer">{{ site.footer_text | markdownify }}</p>
-        <p class="divider footer">{{ site.contact_email_footer | markdownify }}</p>
+        <div id="desktopFooter">
+            <p class="divider footer">{{ site.footer_text | markdownify }}</p>
+            <p class="divider footer">{{ site.contact_email_footer | markdownify }}</p>
+        </div>
```

Comment:
I may try and adjust this. I do want the sticky footer to be that single line text, but still keep the general footer messaging available if possible.

Reply:
+1

Reply:
+1